src/lib/userStorage/UserStorageClass.js

```diff
@@ -605,76 +590,24 @@ export class UserStorage {
     }
   }
 
-  // checkAvatar was removed as we don't need to keep updates/migrations only funcitons in the common API
-
-  async setAvatar(avatar, withCleanup = false) {
-    // save space and load on gun
-    const avatarResized = await resizeImage(avatar, 320)
-
-    // eslint-disable-next-line
-    return Promise.all([
-      this._storeAvatar('avatar', avatarResized, withCleanup),
-      this.setSmallAvatar(avatarResized, withCleanup),
-    ])
-  }
-
-  async setSmallAvatar(avatar, withCleanup = false) {
-    const smallAvatar = await resizeImage(avatar, 50)
-
-    return this._storeAvatar('smallAvatar', smallAvatar, withCleanup)
-  }
-
-  // eslint-disable-next-line require-await
-  async removeAvatar(withCleanup = false) {
-    return Promise.all(
-      // eslint-disable-next-line require-await
-      ['avatar', 'smallAvatar'].map(async field => {
-        // eslint-disable-next-line require-await
-        const updateGunDB = async () => this.setProfileField(field, null, 'public')
-
-        if (true !== withCleanup) {
-          return updateGunDB()
-        }
-
-        return this._removeBase64(field, updateGunDB)
-      }),
-    )
-  }
-
   /**
-   * @private
-   * @param {String} avatar Base64 data url string
-   *
-   * @returns {Promise<string>} CID
+   * Avatar setter
+   * @param avatar
+   * @returns {Promise<CID[]>}
    */
-  async _storeAvatar(field, avatar, withCleanup = false) {
-    const cid = await Base64Storage.store(avatar)
-    // eslint-disable-next-line require-await
-    const updateGunDB = async () => this.setProfileField(field, cid, 'public')
-
-    if (true !== withCleanup) {
-      return updateGunDB()
-    }
-
-    return this._removeBase64(field, updateGunDB)
+  // eslint-disable-next-line require-await
+  async setAvatar(avatar) {
```

Comment:
remove any proxy methods.
from outside they should be accessible via `userStorage.userProfileStorage.setAvatar()` etc
also pls rename `userProfileStorage` to just `.profile` (will be shorter and useful)

Reply:
i'll do that

Reply:
i checked how the feed is implemented. they also using method proxying
so let's keep it with profile methods too